src/components/media-views.js

```diff
@@ -79,19 +79,27 @@ async function createGIFTexture(url) {
  * @param {string} src - Url to a video file.
  * @returns {Element} Video element.
  */
-function createVideoEl(src) {
+async function createVideoEl(src) {
   const videoEl = document.createElement("video");
   videoEl.setAttribute("playsinline", "");
   videoEl.setAttribute("webkit-playsinline", "");
   videoEl.loop = true;
   videoEl.crossOrigin = "anonymous";
-  videoEl.src = src;
+
+  if (!src.startsWith("webrtc://")) {
+    videoEl.src = src;
+  } else {
+    const streamClientId = src.substring(9);
+    const stream = await NAF.connection.adapter.getMediaStream(streamClientId, "video");
+    videoEl.srcObject = new MediaStream(stream.getVideoTracks());
```

Comment:
I would just use the `MediaStream` directly rather than making a new one with just video.

Reply:
this was based on the previous implementation, which had a comment I failed to carry over. will fix